packages/@aws-cdk/aws-eks/README.md

```diff
@@ -253,44 +254,109 @@ cluster.addFargateProfile('MyProfile', {
 });
 ```
 
-To create an EKS cluster that **only** uses Fargate capacity, you can use
-`FargateCluster`.
+> For a complete API reference visit [`FargateProfileOptions`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-eks.FargateProfileOptions.html)
+
+You can also directly use the `FargateProfile` construct to create profiles under different scopes:
+
+```ts
+new eks.FargateProfile(scope, 'MyProfile', {
+  cluster,
+  ...
+});
+```
+
+> For a complete API reference visit [`FargateProfileProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-eks.FargateProfileProps.html)
 
-The following code defines an Amazon EKS cluster without EC2 capacity and a default
-Fargate Profile that matches all pods from the "kube-system" and "default" namespaces. It is also configured to [run CoreDNS on Fargate](https://docs.aws.amazon.com/eks/latest/userguide/fargate-getting-started.html#fargate-gs-coredns) through the `coreDnsComputeType` cluster option.
+To create an EKS cluster that **only** uses Fargate capacity, you can use `FargateCluster`.
+The following code defines an Amazon EKS cluster with a default Fargate Profile that matches all pods from the "kube-system" and "default" namespaces. It is also configured to [run CoreDNS on Fargate](https://docs.aws.amazon.com/eks/latest/userguide/fargate-getting-started.html#fargate-gs-coredns).
 
 ```ts
 const cluster = new eks.FargateCluster(this, 'MyCluster', {
   version: eks.KubernetesVersion.V1_16,
 });
-
- // apply k8s resources on this cluster
-cluster.addManifest(...);
 ```
 
 **NOTE**: Classic Load Balancers and Network Load Balancers are not supported on
 pods running on Fargate. For ingress, we recommend that you use the [ALB Ingress
 Controller](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html)
 on Amazon EKS (minimum version v1.1.4).
 
-### Spot Capacity
+#### 3) Self Managed Auto Scaling Groups
```

Comment:
Decided to use `Self-Managed` terminology in the README since it aligns with the EKS documentation wording.
The API itself doesn't use this term.